.github/workflows/build.yml

```diff
@@ -66,6 +70,17 @@ jobs:
             --api-version 6.5 --system-cxx \
             --target-dir build
           ninja -C src/out/build
+          
+      - name: Build for Tizen 8.0
+        if: ${{ matrix.api-version == 8 }}
+        run: |
+          src/tools/gn \
+            --target-cpu ${{ matrix.arch }} \
+            --target-toolchain /usr/lib/llvm-12 \
+            --target-sysroot src/sysroot-8.0/${{ matrix.arch }} \
+            --api-version 8.0 --system-cxx \
```

Comment:
This will effectively remove NUI support from the 8.0 embedder (see line 181 in `tizen/BUILD.gn`). If we're to compare more than two API versions, it would be better to create a dedicated command line option for each build flavor, e.g. `--[no]-nui`, `--[no]-cbhm`, because it's not quite easy to compare numbers in GN. Let me help if you're not familiar with the build system.